frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -372,6 +374,20 @@ export const VMSchedulingList: React.FC<VMSchedulingListProps> = ({
               ? t('kubevirt-plugin~Workload scheduled with dedicated resources (guaranteed policy)')
               : t('kubevirt-plugin~No Dedicated resources applied')}
           </VMDetailsItem>
+
```

Comment:
should this be added for vm templates aswell? (not sure)
`vm-template-resource.tsx`

Reply:
@yaacov , gilad has a point, does it need to be added to the template as well?

Reply:
YES :+1: the original bug was about templates ...